test/Dialect/FIRRTL/extract-instances.mlir

```diff
@@ -526,14 +539,14 @@ firrtl.circuit "InstSymConflict" {
 firrtl.circuit "Plop_Foo" attributes {annotations = [{class = "sifive.enterprise.firrtl.ExtractClockGatesFileAnnotation", filename = "ckgates.txt", group = "ClockGates"}]} {
   // CHECK: hw.hierpath @nla_1 [@Plop_Foo::@ClockGates, @Plop_ClockGates::@ckg, @EICG_wrapper]
   hw.hierpath @nla_1 [@Plop_Foo::@core, @Plop_Bar::@ckg, @EICG_wrapper]
-  firrtl.extmodule private @EICG_wrapper() attributes {defname = "EICG_wrapper"}
+  firrtl.extmodule private @EICG_wrapper(in in: !firrtl.clock, in test_en: !firrtl.uint<1>, in en: !firrtl.uint<1>, out out: !firrtl.clock) attributes {defname = "EICG_wrapper"}
   firrtl.module private @Plop_Bar() {
-    firrtl.instance ckg sym @ckg @EICG_wrapper()
+    %gate_in, %gate_test_en, %gate_en, %gate_out = firrtl.instance ckg sym @ckg @EICG_wrapper(in in: !firrtl.clock, in test_en: !firrtl.uint<1>, in en: !firrtl.uint<1>, out out: !firrtl.clock)
   }
-  // CHECK: firrtl.module private @Plop_ClockGates()
+  // CHECK: firrtl.module private @Plop_ClockGates(in %clock_gate_0_in: !firrtl.clock, in %clock_gate_0_test_en: !firrtl.uint<1>, in %clock_gate_0_en: !firrtl.uint<1>, out %clock_gate_0_out: !firrtl.clock)
   // CHECK-LABEL: firrtl.module @Plop_Foo()
   firrtl.module @Plop_Foo() attributes {annotations = [{class = "sifive.enterprise.firrtl.MarkDUTAnnotation", prefix = "Plop_"}]} {
-    // CHECK: firrtl.instance ClockGates sym @ClockGates @Plop_ClockGates()
+    // CHECK: %ClockGates_clock_gate_0_in, %ClockGates_clock_gate_0_test_en, %ClockGates_clock_gate_0_en, %ClockGates_clock_gate_0_out = firrtl.instance ClockGates sym @ClockGates @Plop_ClockGates(in clock_gate_0_in: !firrtl.clock, in clock_gate_0_test_en: !firrtl.uint<1>, in clock_gate_0_en: !firrtl.uint<1>, out clock_gate_0_out: !firrtl.clock)
```

Comment:
```suggestion
    // CHECK: firrtl.instance ClockGates sym @ClockGates @Plop_ClockGates
```

This test is really just checking the instance exists where we expect it to, I think this is the right change to make it test what we care about.